src/main/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/ValidationModelService.java

```diff
@@ -24,23 +33,29 @@
     //Note: These are aggregation records from Assignment_history table.
     private static final Logger logger = LoggerFactory.getLogger(ValidationModelService.class);
     private StatelessKieSession kieSession;
-    private IdamRoleService idamRoleService;
+
     private RetrieveDataService retrieveDataService;
-    private SecurityUtils securityUtils;
+
+    private PersistenceService persistenceService;
+
 
     public ValidationModelService(StatelessKieSession kieSession,
-                                  IdamRoleService idamRoleService,
-                                  RetrieveDataService retrieveDataService, SecurityUtils securityUtils) {
+                                  RetrieveDataService retrieveDataService,
+                                  PersistenceService persistenceService) {
         this.kieSession = kieSession;
-        this.idamRoleService = idamRoleService;
```

Comment:
we don't need to remove the idam code, it might be required later so we can still keep it in validation model class.